ui/window.go

```diff
@@ -361,8 +360,8 @@ func (win *Window) handleEvents(gtx C) {
 // handleUserClick listen touch action of user for mobile.
 func (win *Window) handleUserClick(gtx C) {
 	for _, evt := range win.clicker.Events(gtx) {
-		if evt.Type == gesture.TypePress && evt.Source == pointer.Touch {
-			win.load.Theme.AutoHideSoftKeyBoard(gtx)
+		if evt.Type == gesture.TypePress { //&& evt.Source == pointer.Touch {
```

Comment:
Does commenting this out affect mobile users? I'm testing on Desktop atm.

Reply:
it don't affect to mobile users